### PR TITLE
Trace collector default resource request values update

### PR DESCRIPTION
**Pull request by hisarbalik**
**Description**

Changes proposed in this pull request:

- Default values of tracing collector resource request updated

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/15837

### CODE REVIEW

File: resources/telemetry/values.yaml [hunk 1]
```diff
@@ -27,4 +27,3 @@ global:
   log:
     format: "json"
     level: "info"
-  operatorManagedFluentBit: false
```
Comment (skhalash) [inline resources/telemetry/values.yaml:30]: Maybe let's keep this flag?

Comment (a-thaler) [reply resources/telemetry/values.yaml:30]: Might be useful :P

**Review by skhalash — APPROVED** [review]

**Pull request merged**